assets/jquery.arctext.js

```diff
@@ -0,0 +1,381 @@
+/**
+ * Arctext.js
+ * A jQuery plugin for curved text
+ * http://www.codrops.com
+ *
+ * Copyright 2011, Pedro Botelho / Codrops
+ * Free to use under the MIT license.
+ *
+ * Date: Mon Jan 23 2012
+ */
+
+(function( $, undefined ) {
+	
+	/*!	
+	* FitText.js 1.0
+	*
+	* Copyright 2011, Dave Rupert http://daverupert.com
+	* Released under the WTFPL license 
+	* http://sam.zoy.org/wtfpl/
+	*
+	* Date: Thu May 05 14:23:00 2011 -0600
+	*/
+	$.fn.fitText = function( kompressor, options ) {
+
+	    var settings = {
+			'minFontSize' : Number.NEGATIVE_INFINITY,
+			'maxFontSize' : Number.POSITIVE_INFINITY
+		};
+
+		return this.each(function() {
+			var $this = $(this);              
+			var compressor = kompressor || 1; 
+	
+			if ( options ) { 
+			  $.extend( settings, options );
+			}
+	
+			// Resizer() resizes items based on the object width divided by the compressor * 10
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹↹` with `······`](https://app.codacy.com/gh/Anilkum201241/Anilkum201241/pullRequest?prid=10022322)
```suggestion
      // Resizer() resizes items based on the object width divided by the compressor * 10
```